.travis.yml

```diff
@@ -58,5 +58,7 @@ script:
   # in the makefile
   - if [ $REPO_PROJ == "fvp" ]; then mkdir -p $HOME/$REPO_PROJ/Foundation_Platformpkg; fi
   - cd $HOME/$REPO_PROJ && repo init -u https://github.com/OP-TEE/manifest.git -m $REPO_PROJ.xml </dev/null && repo sync -j2 --no-clone-bundle --no-tags --quiet
+  # Fetch a local copy of dtc+libfdt to avoid issues with a possibly outdated libfdt-dev
+  - if [ $REPO_PROJ == "qemu_v8" ]; then cd $HOME/$REPO_PROJ/qemu && git submodule update --init dtc; fi
```

Comment:
Should be OK, but remember that we have to consider two things here. We're using bandwidth from Travis and they have a hard limit of 50 minutes per job. So anything we can do that lowers build time is something we should try to do.